JOSS/paper.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'JLSO: Storing reproducible serialized objects in Julia'
+tags:
+  - julialang
+  - data storage
+  - reproducibility
+authors:
+  - name: Rory Finnegan
+    orcid: 0000-0002-4603-077X
+    affiliation: 1
+  - name: Lyndon White
+    orcid: 0000-0003-1386-1646
+    affiliation: 2
+affiliations:
+ - name: Research Engineer, Invenia Technical Computing
+   index: 1
+ - name: Research Engineer, Invenia Labs
+   index: 2
+date: 24 August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+As scientific computing software grows increasingly complex, the need to efficiently and reliably store sophisticated program objects has become a growing need. The expanding list of file formats for serializing objects is evidence of this problem. Unfortunately, these file formats typically come with usability versus reliability tradeoffs.
```

Comment:
```suggestion
As scientific computing software grows increasingly complex, the need to efficiently and reliably store sophisticated program objects likewise increases.
The expanding list of file formats for serializing objects is evidence of this problem.
Unfortunately, these file formats typically come with usability versus reliability tradeoffs.
```

First sentence seem repeditive.

Also, i think 1 sentence per line is best here (like in docs) so that diffs stay sensible.
I am not going to go an suggest changes everywhere for that.

Reply:
Alright, I think all the sentences are on separate lines now. I've also rewritten that first sentence to make it less repetitive.